content/en/docs/contribute/style/style-guide.md

```diff
@@ -236,6 +235,36 @@ The output is:
 You can _still_ use Markdown inside these callouts.
 {{< /note >}}
 
+You can use a {{</* note */>}} in a list:
+
+```
+1. Use the note shortcode in a list
+
+1. A second item with an embedded note
```

Comment:
was this meant to be 2. and so on ?

Reply:
Hello @DanyC97, my mistake. Should be 1's as this text is demonstrating how to write a list.